app/src/main/java/diemmy/edu/recycler/ItemView.java

```diff
@@ -0,0 +1,29 @@
+package diemmy.edu.recycler;
+
+/**
+ * Created by DELL on 10/10/2017.
+ */
+
+public class ItemView {
+    private String name;
```

Comment:
check framgia convention 
2.1 Java Style Rules

Field names need to be defined at first and follow this conventions.

Non-public, non-static field names start with m.
Static field names start with s.
Other fields start with a lower case letter.
Public static final fields (constants) are ALL_CAPS_WITH_UNDERSCORES.